shell.py

```diff
@@ -245,12 +251,19 @@ class NextTurn(Command):
 
     keywords = ['next']
 
+    @manager.registry.add_binding(Keys.ControlN)
     def do_command(self, *args):
-        if not self.game.tm:
+        if not game.tm:
```

Comment:
How the heck does this even work without the self?? This feels way too magical.

Reply:
Python :roll_eyes: haha 😆